README.md

```diff
@@ -136,6 +137,47 @@ Example below:
 ...
 ```
 
+#### <a name="custom-verification-handler"></a> Custom Verification Handler
+
+A custom verification handler can be called by the PxModule instead of the default behavior, to allow a user to customize the behavior based on the risk score returned by PerimeterX.
+
+The custom handler class should implement the `ICustomVerificationHandler` interface, and its name should be added to the configuration section:   
+
+```xml
+...
+  customVerificationHandler="UniqueVerificationHandler"
+...
+```
+
+The custom logic will reside in the `Handle` method, making use of the following arguments:
+
+- `HttpApplication application` - The currently running ASP.NET application methods, properties and events. Calling [`application.CompleteRequest`](https://msdn.microsoft.com/en-us/library/system.web.httpapplication.completerequest(v=vs.110).aspx), for example, will directly execute the [`EndRequest`](https://msdn.microsoft.com/en-us/library/system.web.httpapplication.endrequest(v=vs.110).aspx) event to return a response to the client.
+- `PxContext pxContext` - The PerimeterX context, containing valuable fields such as `Score`, `UUID`, `BlockAction` etc. 
+- `PxModuleConfigurationSection pxConfig` - The current configuration used by the PxModule, representing the `PerimeterX.PxModuleConfigurationSection` settings. Contains fields such as `BlockingScore`.
+
+Common customization options are presenting of a captcha or a custom branded block page. 
+
+```xml
+...
+
+namespace myUniqueApp
+{
+    public class UniqueVerificationHandler : ICustomVerificationHandler
+    {
+        public void Handle(HttpApplication application, PxContext pxContext, PxModuleConfigurationSection pxConfig)
+        {
+            // Custom verification logic goes here, e.g -
+            // 
+            // if (pxContext.Score >= pxConfig.BlockingScore) // In the case of a high score, present the standard block/captcha page
```

Comment:
why is this code example commented out?

Reply:
I can uncomment it (assuming that the comment in the first line above is enough to indicate that this is no more than an example and not necessarily part of the expected custom logic).

Reply:
I don't think the commented out code explains that this is an example anyway. But maybe change the line above to something like `Custom verification logic goes here. The following code is only an example of a possible implementation.`

Reply:
👍